lib/common/email.sh

```diff
@@ -72,18 +79,21 @@ export -f _get_uploader_rsync
 # $1 - file uploaded
 _get_uploader() {
     local file=$1; shift
+    log_info "File '$file'"
     local uploader=""
 
     uploader=`_get_uploader_ftp $file` || \
         uploader=`_get_uploader_rsync $file`
 
+    log_info "Uploader '$uploader'"
     echo $uploader
 }
 export -f _get_uploader
 
 # returns the email lookup file which maps between usernames and their email
 # addresses
 _email_lookup_file() {
+    log_info "EMAIL_ALIASES '$EMAIL_ALIASES'"
```

Comment:
This is not required as it is just the contents of a file, so no point logging it (just look at the file).